src/components/Timer.jsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+const StyledH1 = styled.h1`
+	font-size: 1.5cm;
+`;
+
+export default function Timer({minutes, seconds}) {
+	return (
+		<StyledH1>
+			{minutes.toString().padStart(2, '0')}:{seconds.toString().padStart(2, '0')}
+		</StyledH1>
```

Comment:
I would recommend to not use an `h1` here as there always should be only one `h1` per page. For example, you import the `Timer` component in the `ExerciseForm.js` where you already have an `h1` element.